### PR TITLE
[Issue 12] Notify any Polymer elements when cell has resized

### PR DESCRIPTION
**Pull request by jhpedemonte**
Listen for and track Polymer elements using the
`iron-resizable-behavior`, then notify them if their parent cell has
been resized.

(c) Copyright IBM Corp. 2015

### CODE REVIEW

**Comment by jhpedemonte** [comment]
Fix for issue #12

File: urth_dash_js/notebook/dashboard-view/dashboard.js [hunk 1]
```diff
@@ -37,6 +37,9 @@ define([
     var cssLoaded = false;
     var idCounter = 0;
 
+    // duration => the resize transition on gridstack and gridstack cells
+    var RESIZE_DURATION = 350;
```
Comment (parente) [inline urth_dash_js/notebook/dashboard-view/dashboard.js]: Is this hardcoded in the gridstack code too? Any way to read from there?

Comment (dalogsdon) [inline urth_dash_js/notebook/dashboard-view/dashboard.js]: This is necessary due to [this bug](https://github.com/troolee/gridstack.js/issues/159) in gridstack where a `resizestop` event is fired while a CSS transition is taking place. 

The value is not hardcoded in the gridstack code; rather it is defined in `gridstack.css` as a `transition-duration`. There is a 300ms CSS transition on cell width/height that we are waiting to complete before taking action based on the size of a cell after resizing.

**Comment by jhpedemonte** [comment]
@parente: We verified these changes using webgl-globe. This is ready to be merged.

**Pull request merged**